client/src/app/core/repositories/agenda/list-of-speakers-repository.service.ts

```diff
@@ -29,9 +31,12 @@ export interface SpeakingTimeStructureLevelObject {
 @Injectable({
     providedIn: 'root'
 })
-export class ListOfSpeakersRepositoryService extends MeetingModelBaseRepository<ViewListOfSpeakers, ListOfSpeakers> {
-    public constructor(repositoryServiceCollector: RepositoryServiceCollector) {
-        super(repositoryServiceCollector, ListOfSpeakers);
+export class ListOfSpeakersRepositoryService extends MeetingModelWithErrorRepository<
+    ViewListOfSpeakers,
+    ListOfSpeakers
+> {
+    public constructor(repositoryServiceCollector: RepositoryServiceCollector, errorService: ErrorService) {
```

Comment:
You can remove the ctor, if it just calls super ans passes through the arguments I believe.